<!--REQUEST_ID=e3e60dd2-2cd6-491a-bb31-e869b8a9580a-->
Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.45995&attempt_id=46e84562-2f9b-4fab-bbf6-f4ce2cd34a93)

Attempted: gede

<details><summary>Partial log (click to expand)</summary><p>

```
3 errors generated.
make: *** [Makefile:867: core.o] Error 1
Checking for make...  found in /nix/store/qfag24z6xsr6jkyi8gb4cv62rp945rbk-gnumake-4.2.1/bin
Checking for gcc...  not found!!
Checking for ctags...  found in /nix/store/r4jzzs35n0rlvahs2k2hbhzmpvhf2bva-ctags-816/bin
Detecting Qt version... Qt? found (qmake)
Generating makefile
Compiling (please wait)
builder for '/nix/store/66k9c9ygfn7hi42rjfillaqa72fyn1r3-gede-2.10.9.drv' failed with exit code 1
error: build of '/nix/store/66k9c9ygfn7hi42rjfillaqa72fyn1r3-gede-2.10.9.drv' failed
```
</p></details>

--- a/pkgs/development/tools/misc/gede/default.nix
+++ b/pkgs/development/tools/misc/gede/default.nix
@@ -2,11 +2,11 @@
 
 stdenv.mkDerivation rec {
   name = "gede-${version}";
-  version = "2.6.1";
+  version = "2.10.9";
 
   src = fetchurl {
     url = "http://gede.acidron.com/uploads/source/${name}.tar.xz";
-    sha256 = "0jallpchl3c3i90hwic4n7n0ggk5wra0fki4by9ag26ln0k42c4r";
+    sha256 = "0av9v3r6x6anjjm4hzn8wxnvrqc8zp1g7570m5ndg7cgc3sy3bg6";
   };
 
   nativeBuildInputs = [ qmake makeWrapper python ];
@@ -27,7 +27,7 @@ stdenv.mkDerivation rec {
     description = "Graphical frontend (GUI) to GDB";
     homepage = http://gede.acidron.com;
     license = licenses.bsd2;
-    platforms = platforms.unix;
+    platforms = platforms.linux;
     maintainers = with maintainers; [ juliendehos ];
   };
 }

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -8264,7 +8264,7 @@ with pkgs;
 
   funnelweb = callPackage ../development/tools/literate-programming/funnelweb { };
 
-  gede = libsForQt5.callPackage ../development/tools/misc/gede { };
+  gede = libsForQt59.callPackage ../development/tools/misc/gede { };
 
   gdbgui = callPackage ../development/tools/misc/gdbgui { };
 

